flux-pipe/plotting/plot_fieldmap.py

```diff
@@ -0,0 +1,280 @@
+
+
+# datdir = "/Users/cgilbert/vscode/fluxon-data/"
```

Comment:
Sometimes a good idea (I'm bad about this), of not pointing towards local directories. But this is a comment, so it could probably go anyway.

Reply:
I intend to replace all this garbage with a function that loads an environment variable... Isn't there already one that points to the flux directory?

Reply:
I have a few that point towards local perl directories / the compiled FLUX libraries. Sometimes I write code that assumes the user is in a directory they want to use as a working directory for that project - not always the code repository itself. If we need any environmental variables, we should also make sure to update any relevant bits in the readme install guide.

Reply:
depricated code + will fix everything at the same time in the future